## Travis tests have failed

Hey @utkarsh2102,
Please read the following log in order to understand the failure reason.
It'll be awesome if you fix what's wrong and commit the changes.


### 1st Build

<a href="https:&#x2F;&#x2F;travis-ci.org&#x2F;coala&#x2F;gci-leaders&#x2F;jobs&#x2F;445064477">View build log</a>


<details>
  <summary>
    <strong>
     coala --non-interactive -V
    </strong>
  </summary>

```
[WARNING][11:18:58] StyleLintBear: Using unnecessary capturing groups affects the performance of coala. You should use '(?:<pattern>)' instead of '(<pattern>)' for your regex.
[DEBUG][11:18:59] Platform Linux -- Python 3.6.3, coalib 0.11.0
Executing section all...
[DEBUG][11:19:02] Files that will be checked:
/home/travis/build/coala/gci-leaders/webpack.config.js
/home/travis/build/coala/gci-leaders/tests/lib/pre2017.test.js
/home/travis/build/coala/gci-leaders/tests/lib/generate.test.js
/home/travis/build/coala/gci-leaders/tests/lib/gather.test.js
/home/travis/build/coala/gci-leaders/tests/lib/scrape.test.js
/home/travis/build/coala/gci-leaders/tests/lib/rss.test.js
/home/travis/build/coala/gci-leaders/tests/lib/planet.test.js
/home/travis/build/coala/gci-leaders/lib/generate.js
/home/travis/build/coala/gci-leaders/lib/planet.js
/home/travis/build/coala/gci-leaders/lib/utils.js
/home/travis/build/coala/gci-leaders/lib/queries/index.js
/home/travis/build/coala/gci-leaders/lib/scrape.js
/home/travis/build/coala/gci-leaders/lib/rss.js
/home/travis/build/coala/gci-leaders/lib/pre2017.js
/home/travis/build/coala/gci-leaders/lib/main.js
/home/travis/build/coala/gci-leaders/lib/gather.js
/home/travis/build/coala/gci-leaders/src/js/utils.js
/home/travis/build/coala/gci-leaders/src/js/locale.js
/home/travis/build/coala/gci-leaders/src/js/app.js
/home/travis/build/coala/gci-leaders/src/js/twitter.js
/home/travis/build/coala/gci-leaders/src/js/bootstrap.js
/home/travis/build/coala/gci-leaders/package.json
/home/travis/build/coala/gci-leaders/static/i18n/es.json
/home/travis/build/coala/gci-leaders/static/i18n/pl.json
/home/travis/build/coala/gci-leaders/static/i18n/nb_NO.json
/home/travis/build/coala/gci-leaders/static/i18n/en.json
/home/travis/build/coala/gci-leaders/src/styles/main.css
/home/travis/build/coala/gci-leaders/.travis.yml
/home/travis/build/coala/gci-leaders/templates/planet.html
/home/travis/build/coala/gci-leaders/templates/main.html
/home/travis/build/coala/gci-leaders/templates/pre2017.html
/home/travis/build/coala/gci-leaders/.eslintrc
[DEBUG][11:19:02] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
Executing section all.whitespace...
[DEBUG][11:19:02] Files that will be checked:
/home/travis/build/coala/gci-leaders/webpack.config.js
/home/travis/build/coala/gci-leaders/tests/lib/pre2017.test.js
/home/travis/build/coala/gci-leaders/tests/lib/generate.test.js
/home/travis/build/coala/gci-leaders/tests/lib/gather.test.js
/home/travis/build/coala/gci-leaders/tests/lib/scrape.test.js
/home/travis/build/coala/gci-leaders/tests/lib/rss.test.js
/home/travis/build/coala/gci-leaders/tests/lib/planet.test.js
/home/travis/build/coala/gci-leaders/lib/generate.js
/home/travis/build/coala/gci-leaders/lib/planet.js
/home/travis/build/coala/gci-leaders/lib/utils.js
/home/travis/build/coala/gci-leaders/lib/queries/index.js
/home/travis/build/coala/gci-leaders/lib/scrape.js
/home/travis/build/coala/gci-leaders/lib/rss.js
/home/travis/build/coala/gci-leaders/lib/pre2017.js
/home/travis/build/coala/gci-leaders/lib/main.js
/home/travis/build/coala/gci-leaders/lib/gather.js
/home/travis/build/coala/gci-leaders/src/js/utils.js
/home/travis/build/coala/gci-leaders/src/js/locale.js
/home/travis/build/coala/gci-leaders/src/js/app.js
/home/travis/build/coala/gci-leaders/src/js/twitter.js
/home/travis/build/coala/gci-leaders/src/js/bootstrap.js
/home/travis/build/coala/gci-leaders/package.json
/home/travis/build/coala/gci-leaders/static/i18n/es.json
/home/travis/build/coala/gci-leaders/static/i18n/pl.json
/home/travis/build/coala/gci-leaders/static/i18n/nb_NO.json
/home/travis/build/coala/gci-leaders/static/i18n/en.json
/home/travis/build/coala/gci-leaders/src/styles/main.css
/home/travis/build/coala/gci-leaders/.travis.yml
/home/travis/build/coala/gci-leaders/templates/planet.html
/home/travis/build/coala/gci-leaders/templates/main.html
/home/travis/build/coala/gci-leaders/templates/pre2017.html
/home/travis/build/coala/gci-leaders/.eslintrc
[DEBUG][11:19:03] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
[DEBUG][11:19:03] Running bear SpaceConsistencyBear...
Executing section all.linelength...
[WARNING][11:19:03] No files matching '/home/travis/build/coala/gci-leaders/**.json' were found.
[WARNING][11:19:03] No files matching '/home/travis/build/coala/gci-leaders/**.html' were found.
[DEBUG][11:19:03] Files that will be checked:
/home/travis/build/coala/gci-leaders/webpack.config.js
/home/travis/build/coala/gci-leaders/tests/lib/pre2017.test.js
/home/travis/build/coala/gci-leaders/tests/lib/generate.test.js
/home/travis/build/coala/gci-leaders/tests/lib/gather.test.js
/home/travis/build/coala/gci-leaders/tests/lib/scrape.test.js
/home/travis/build/coala/gci-leaders/tests/lib/rss.test.js
/home/travis/build/coala/gci-leaders/tests/lib/planet.test.js
/home/travis/build/coala/gci-leaders/lib/generate.js
/home/travis/build/coala/gci-leaders/lib/planet.js
/home/travis/build/coala/gci-leaders/lib/utils.js
/home/travis/build/coala/gci-leaders/lib/queries/index.js
/home/travis/build/coala/gci-leaders/lib/scrape.js
/home/travis/build/coala/gci-leaders/lib/rss.js
/home/travis/build/coala/gci-leaders/lib/pre2017.js
/home/travis/build/coala/gci-leaders/lib/main.js
/home/travis/build/coala/gci-leaders/lib/gather.js
/home/travis/build/coala/gci-leaders/src/js/utils.js
/home/travis/build/coala/gci-leaders/src/js/locale.js
/home/travis/build/coala/gci-leaders/src/js/app.js
/home/travis/build/coala/gci-leaders/src/js/twitter.js
/home/travis/build/coala/gci-leaders/src/js/bootstrap.js
/home/travis/build/coala/gci-leaders/src/styles/main.css
/home/travis/build/coala/gci-leaders/.travis.yml
/home/travis/build/coala/gci-leaders/.eslintrc
[DEBUG][11:19:03] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
[DEBUG][11:19:03] Running bear LineLengthBear...
Executing section all.links...
[DEBUG][11:19:03] Files that will be checked:
/home/travis/build/coala/gci-leaders/webpack.config.js
/home/travis/build/coala/gci-leaders/tests/lib/pre2017.test.js
/home/travis/build/coala/gci-leaders/tests/lib/generate.test.js
/home/travis/build/coala/gci-leaders/tests/lib/gather.test.js
/home/travis/build/coala/gci-leaders/tests/lib/scrape.test.js
/home/travis/build/coala/gci-leaders/tests/lib/rss.test.js
/home/travis/build/coala/gci-leaders/tests/lib/planet.test.js
/home/travis/build/coala/gci-leaders/lib/generate.js
/home/travis/build/coala/gci-leaders/lib/planet.js
/home/travis/build/coala/gci-leaders/lib/utils.js
/home/travis/build/coala/gci-leaders/lib/queries/index.js
/home/travis/build/coala/gci-leaders/lib/scrape.js
/home/travis/build/coala/gci-leaders/lib/rss.js
/home/travis/build/coala/gci-leaders/lib/pre2017.js
/home/travis/build/coala/gci-leaders/lib/main.js
/home/travis/build/coala/gci-leaders/lib/gather.js
/home/travis/build/coala/gci-leaders/src/js/utils.js
/home/travis/build/coala/gci-leaders/src/js/locale.js
/home/travis/build/coala/gci-leaders/src/js/app.js
/home/travis/build/coala/gci-leaders/src/js/twitter.js
/home/travis/build/coala/gci-leaders/src/js/bootstrap.js
/home/travis/build/coala/gci-leaders/package.json
/home/travis/build/coala/gci-leaders/static/i18n/es.json
/home/travis/build/coala/gci-leaders/static/i18n/pl.json
/home/travis/build/coala/gci-leaders/static/i18n/nb_NO.json
/home/travis/build/coala/gci-leaders/static/i18n/en.json
/home/travis/build/coala/gci-leaders/src/styles/main.css
/home/travis/build/coala/gci-leaders/.travis.yml
/home/travis/build/coala/gci-leaders/templates/planet.html
/home/travis/build/coala/gci-leaders/templates/main.html
/home/travis/build/coala/gci-leaders/templates/pre2017.html
/home/travis/build/coala/gci-leaders/.eslintrc
[DEBUG][11:19:03] Starting new HTTP connection (1): 216.58.218.174:80
[DEBUG][11:19:03] http://216.58.218.174:80 "HEAD / HTTP/1.1" 301 0
[DEBUG][11:19:03] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Starting new HTTPS connection (1): raw.githubusercontent.com:443
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Running bear InvalidLinkBear...
[DEBUG][11:19:03] Starting new HTTPS connection (1): codein.withgoogle.com:443
[DEBUG][11:19:04] https://codein.withgoogle.com:443 "HEAD /api HTTP/1.1" 301 0
[DEBUG][11:19:04] https://raw.githubusercontent.com:443 "HEAD / HTTP/1.1" 301 0
[DEBUG][11:19:04] Starting new HTTPS connection (1): codein.withgoogle.com:443
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Starting new HTTPS connection (1): www.travisbuddy.com:443
[DEBUG][11:19:04] https://codein.withgoogle.com:443 "HEAD /api/program/current/ HTTP/1.1" 200 0
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Starting new HTTPS connection (1): codein.withgoogle.com:443
[DEBUG][11:19:04] https://codein.withgoogle.com:443 "HEAD /archive/2016 HTTP/1.1" 301 0
[DEBUG][11:19:04] Starting new HTTPS connection (1): codein.withgoogle.com:443
[DEBUG][11:19:04] https://codein.withgoogle.com:443 "HEAD /archive/2015 HTTP/1.1" 301 0
[DEBUG][11:19:04] Starting new HTTPS connection (1): en.wikipedia.org:443
[DEBUG][11:19:04] https://en.wikipedia.org:443 "HEAD /wiki/Google_Code-in HTTP/1.1" 200 0
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Running bear InvalidLinkBear...
[DEBUG][11:19:04] Starting new HTTP connection (1): codein.withgoogle.com:80
[DEBUG][11:19:04] http://codein.withgoogle.com:80 "HEAD / HTTP/1.1" 302 0
[DEBUG][11:19:04] Starting new HTTPS connection (1): twitter.com:443
[DEBUG][11:19:04] https://twitter.com:443 "HEAD /hashtag/googlecodein HTTP/1.1" 302 0
[DEBUG][11:19:04] Starting new HTTPS connection (1): gitlab.com:443
[DEBUG][11:19:04] https://gitlab.com:443 "HEAD /gitlab-com/gitlab-artwork/blob/master/LICENSE HTTP/1.1" 200 0
[DEBUG][11:19:04] Starting new HTTPS connection (1): coala.io:443
[DEBUG][11:19:05] https://coala.io:443 "HEAD / HTTP/1.1" 200 0
[DEBUG][11:19:05] Starting new HTTPS connection (1): gci-leaders.netlify.com:443
[DEBUG][11:19:05] Running bear InvalidLinkBear...
[DEBUG][11:19:05] Running bear InvalidLinkBear...
[DEBUG][11:19:06] https://gci-leaders.netlify.com:443 "HEAD / HTTP/1.1" 200 0
[DEBUG][11:19:06] Starting new HTTPS connection (1): gci-leaders.netlify.com:443
[DEBUG][11:19:06] https://gci-leaders.netlify.com:443 "HEAD / HTTP/1.1" 200 0
[DEBUG][11:19:06] Starting new HTTPS connection (1): g.co:443
[DEBUG][11:19:06] https://g.co:443 "HEAD /gci HTTP/1.1" 301 0
[DEBUG][11:19:06] Starting new HTTPS connection (1): g.co:443
[DEBUG][11:19:06] https://g.co:443 "HEAD /gci HTTP/1.1" 301 0
[DEBUG][11:19:06] Running bear InvalidLinkBear...
[WARNING][11:19:19] InvalidLinkBear: This result has no patch attached.

.travis.yml
|  18| ··webhooks:·https://www.travisbuddy.com/
|    | [MAJOR] InvalidLinkBear:
|    | Broken link - unable to connect to https://www.travisbuddy.com/
Executing section js...
[DEBUG][11:19:19] Files that will be checked:
/home/travis/build/coala/gci-leaders/lib/generate.js
/home/travis/build/coala/gci-leaders/lib/planet.js
/home/travis/build/coala/gci-leaders/lib/utils.js
/home/travis/build/coala/gci-leaders/lib/scrape.js
/home/travis/build/coala/gci-leaders/lib/rss.js
/home/travis/build/coala/gci-leaders/lib/pre2017.js
/home/travis/build/coala/gci-leaders/lib/main.js
/home/travis/build/coala/gci-leaders/lib/gather.js
/home/travis/build/coala/gci-leaders/lib/queries/index.js
/home/travis/build/coala/gci-leaders/src/js/utils.js
/home/travis/build/coala/gci-leaders/src/js/locale.js
/home/travis/build/coala/gci-leaders/src/js/app.js
/home/travis/build/coala/gci-leaders/src/js/twitter.js
/home/travis/build/coala/gci-leaders/src/js/bootstrap.js
[DEBUG][11:19:19] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:19] Running bear ESLintBear...
[DEBUG][11:19:19] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/src/js/twitter.js --config .eslintrc'
[DEBUG][11:19:19] Running bear ESLintBear...
[DEBUG][11:19:19] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/scrape.js --config .eslintrc'
[DEBUG][11:19:20] Running bear ESLintBear...
[DEBUG][11:19:20] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/src/js/app.js --config .eslintrc'
[DEBUG][11:19:21] Use of change_line method is deprecated. Instead use modify_line method, without the original_line argument
[DEBUG][11:19:21] Use of change_line method is deprecated. Instead use modify_line method, without the original_line argument
[DEBUG][11:19:21] Running bear ESLintBear...

lib/scrape.js
[DEBUG][11:19:21] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/queries/index.js --config .eslintrc'
| 106| ··const·res·=·await·fetch
|    | [MAJOR] ESLintBear (prettier/prettier):
|    | Insert `(`

lib/scrape.js
| 107| ······(`${GCI_API_BASE}/program/${GCI_YEAR}/organization/?status=2`)
|    | [MAJOR] ESLintBear (prettier/prettier):
|    | Replace `··(`${GCI_API_BASE}/program/${GCI_YEAR}/organization/?status=2`` with ``${GCI_API_BASE}/program/${GCI_YEAR}/organization/?status=2`⏎··`
[DEBUG][11:19:21] Running bear ESLintBear...
[DEBUG][11:19:21] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/pre2017.js --config .eslintrc'
[DEBUG][11:19:22] Running bear ESLintBear...
[DEBUG][11:19:22] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/generate.js --config .eslintrc'
[DEBUG][11:19:22] Running bear ESLintBear...
[DEBUG][11:19:22] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/utils.js --config .eslintrc'
[DEBUG][11:19:23] Running bear ESLintBear...
[DEBUG][11:19:23] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/gather.js --config .eslintrc'
[DEBUG][11:19:23] Running bear ESLintBear...
[DEBUG][11:19:23] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/planet.js --config .eslintrc'
[DEBUG][11:19:24] Running bear ESLintBear...
[DEBUG][11:19:24] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/src/js/locale.js --config .eslintrc'
[DEBUG][11:19:25] Running bear ESLintBear...
[DEBUG][11:19:25] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/src/js/bootstrap.js --config .eslintrc'
[DEBUG][11:19:25] Running bear ESLintBear...
[DEBUG][11:19:25] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/rss.js --config .eslintrc'
[DEBUG][11:19:26] Running bear ESLintBear...
[DEBUG][11:19:26] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/lib/main.js --config .eslintrc'
[DEBUG][11:19:26] Running bear ESLintBear...
[DEBUG][11:19:26] Running 'eslint --no-ignore --no-color -f=json --stdin --stdin-filename=/home/travis/build/coala/gci-leaders/src/js/utils.js --config .eslintrc'
Executing section css...
[DEBUG][11:19:28] Files that will be checked:
/home/travis/build/coala/gci-leaders/src/styles/main.css
[DEBUG][11:19:28] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:28] Running bear StyleLintBear...
[DEBUG][11:19:28] Running 'stylelint /home/travis/build/coala/gci-leaders/src/styles/main.css --config=/tmp/tmpsaauhbn1.json'
Executing section html...
[DEBUG][11:19:29] Files that will be checked:
/home/travis/build/coala/gci-leaders/templates/planet.html
/home/travis/build/coala/gci-leaders/templates/main.html
/home/travis/build/coala/gci-leaders/templates/pre2017.html
[DEBUG][11:19:29] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:29] Running bear HTMLLintBear...
[DEBUG][11:19:29] Running '/home/travis/virtualenv/python3.6.3/bin/python /home/travis/virtualenv/python3.6.3/bin/html_lint.py --disable=optional /home/travis/build/coala/gci-leaders/templates/planet.html'
[DEBUG][11:19:29] Running bear HTMLLintBear...
[DEBUG][11:19:29] Running '/home/travis/virtualenv/python3.6.3/bin/python /home/travis/virtualenv/python3.6.3/bin/html_lint.py --disable=optional /home/travis/build/coala/gci-leaders/templates/pre2017.html'
[DEBUG][11:19:29] Running bear HTMLLintBear...
[DEBUG][11:19:29] Running '/home/travis/virtualenv/python3.6.3/bin/python /home/travis/virtualenv/python3.6.3/bin/html_lint.py --disable=optional /home/travis/build/coala/gci-leaders/templates/main.html'
Executing section all.json-i18n...
[DEBUG][11:19:29] Files that will be checked:
/home/travis/build/coala/gci-leaders/static/i18n/es.json
/home/travis/build/coala/gci-leaders/static/i18n/pl.json
/home/travis/build/coala/gci-leaders/static/i18n/nb_NO.json
/home/travis/build/coala/gci-leaders/static/i18n/en.json
[DEBUG][11:19:29] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:29] Running bear JSONFormatBear...
[DEBUG][11:19:29] Running bear JSONFormatBear...
[DEBUG][11:19:29] Running bear JSONFormatBear...
[DEBUG][11:19:29] Running bear JSONFormatBear...
Executing section all.json...
[DEBUG][11:19:30] Files that will be checked:
/home/travis/build/coala/gci-leaders/package.json
[DEBUG][11:19:30] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:30] Running bear JSONFormatBear...
Executing section yaml...
[DEBUG][11:19:30] Files that will be checked:
/home/travis/build/coala/gci-leaders/.travis.yml
/home/travis/build/coala/gci-leaders/.eslintrc
[DEBUG][11:19:30] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:30] Running bear YAMLLintBear...
[DEBUG][11:19:30] Running 'yamllint -f parsable /home/travis/build/coala/gci-leaders/.eslintrc --config-file=/tmp/tmp_lmcmd1o'
[DEBUG][11:19:30] Running bear YAMLLintBear...
[DEBUG][11:19:30] Running 'yamllint -f parsable /home/travis/build/coala/gci-leaders/.travis.yml --config-file=/tmp/tmp6r0we_bo'
Executing section commit...
[DEBUG][11:19:30] Files that will be checked:
[nltk_data] Downloading package punkt to /home/travis/nltk_data...
[nltk_data]   Unzipping tokenizers/punkt.zip.
[nltk_data] Downloading package maxent_treebank_pos_tagger to
[nltk_data]     /home/travis/nltk_data...
[nltk_data]   Unzipping taggers/maxent_treebank_pos_tagger.zip.
[nltk_data] Downloading package averaged_perceptron_tagger to
[nltk_data]     /home/travis/nltk_data...
[nltk_data]   Unzipping taggers/averaged_perceptron_tagger.zip.
[DEBUG][11:19:33] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][11:19:33] Running bear GitCommitBear...
[WARNING][11:19:33] Bear GitCommitBear failed to run. Take a look at debug messages (`-V`) for further information.
[DEBUG][11:19:33] The bear GitCommitBear raised an exception. If you are the author of this bear, please make sure to catch all exceptions. If not and this error annoys you, you might want to get in contact with the author of this bear.

Traceback information is provided below:

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/coalib/bears/Bear.py", line 282, in execute
    return [] if result is None else list(result)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/bears/vcs/git/GitCommitBear.py", line 135, in run
    **self.get_issue_checks_metadata().filter_parameters(kwargs))
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/bears/vcs/git/GitCommitBear.py", line 303, in check_issue_reference
    body = body.splitlines()[-1]
IndexError: list index out of range
Executing section cli...
[DEBUG][11:19:34] Files that will be checked:
[DEBUG][11:19:34] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
```

</details>


###### TravisBuddy Request Identifier: a2df2dc0-d6b5-11e8-a0c1-37e29dcbd83a

--- a/lib/generate.js
+++ b/lib/generate.js
@@ -8,6 +8,7 @@ const dates = require('../out/dates.json')
 const planet = require('../out/blog_planet.json')
 const manifest = require('../out/manifest.json')
 const pre2017 = require('../out/pre2017.json')
+const { year } = require('../out/year.json')
 
 const time = fs.statSync(`${__dirname}/../out/data.json`).mtime
 const datetime = new Date(time).toUTCString()
@@ -35,6 +36,7 @@ fs.writeFileSync(
   Mustache.render(fs.readFileSync('templates/main.html').toString(), {
     withLeader,
     datetime,
+    year,
     rootURL,
     noLeader,
     competitionOpen,

--- a/lib/scrape.js
+++ b/lib/scrape.js
@@ -95,15 +95,31 @@ try {
 } catch (e) {
   console.log('No existing data...')
 }
-
+let year = new Date().getFullYear()
 async function fetchProgram() {
-  const res = await fetch(`${GCI_API_BASE}/program/2017/`)
-  return await res.json()
+  const res = await fetch(`${GCI_API_BASE}/program/${year}/`)
+  if (res.status == 200) {
+    return await res.json()
+  }
+
+  year -= 1
+  const lastYearApi = `${GCI_API_BASE}/program/${year}/`
+  return await (await fetch(lastYearApi)).json()
 }
 
 async function fetchOrgs() {
-  const res = await fetch(`${GCI_API_BASE}/program/2017/organization/?status=2`)
-  const { results } = await res.json()
+  const res = await fetch(
+    `${GCI_API_BASE}/program/${year}/organization/?status=2`
+  )
+
+  if (res.status == 200) {
+    const { results } = await res.json()
+    return results
+  }
+  year -= 1
+  const lastYearApi = `${GCI_API_BASE}/program/${year}/`
+  const { results } = await (await fetch(lastYearApi)).json()
+
   return results
 }
 
@@ -654,6 +670,10 @@ async function fetchDates() {
   fs.writeFileSync(
     `${__dirname}/../out/data.json`,
     JSON.stringify(data, null, 2)
+  )
+  fs.writeFileSync(
+    `${__dirname}/../out/year.json`,
+    JSON.stringify({ year: year })
   )
   // minified JSON
   fs.writeFileSync(`${__dirname}/../out/data.min.json`, JSON.stringify(data))

--- a/static/i18n/en.json
+++ b/static/i18n/en.json
@@ -1,5 +1,5 @@
 {
-    "page-title": "Google Code-in 2017 Current Leaders",
+    "page-title": "Google Code-in 2018 Current Leaders",
     "last-updated": "Last updated",
     "order-info": "The leading participants for each organization (and their GitHub accounts, if applicable) are listed randomly.",
     "tasks-completed": "Tasks Completed",

--- a/static/i18n/es.json
+++ b/static/i18n/es.json
@@ -1,5 +1,5 @@
 {
-    "page-title": "Líderes actuales de Google Code-in 2017",
+    "page-title": "Líderes actuales de Google Code-in 2018",
     "last-updated": "Última actualización",
     "order-info": "Los participantes líderes de cada organización (y sus respectivas cuentas de GitHub) están listados de forma aleatoria.",
     "tasks-completed": "Tareas completadas",

--- a/static/i18n/pl.json
+++ b/static/i18n/pl.json
@@ -1,5 +1,5 @@
 {
-    "page-title": "Obecni liderzy Google Code-in 2017",
+    "page-title": "Obecni liderzy Google Code-in 2018",
     "last-updated": "Ostatnio aktualizowano",
     "order-info": "Uczestnicy na prowadzeniu z każdej organizacji (i ich konta na GitHubie, jeśli dotyczy) wypisani są w kolejności losowej.",
     "tasks-completed": "Ukończone zadania",

--- a/templates/main.html
+++ b/templates/main.html
@@ -5,7 +5,7 @@
     <title>GCI Leaders</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta property="og:title" content="GCI Current Leaders">
-    <meta property="og:description" content="Google Code-in 2017 Current Leaders">
+    <meta property="og:description" content="Google Code-in {{year}} Current Leaders">
     <meta property="og:image" content="{{{rootURL}}}/images/logos/gcilogo.png">
     <link rel="stylesheet" href="{{assets.main-css}}">
     <style>.github-corner:hover .octo-arm{animation:octocat-wave 560ms ease-in-out}@keyframes octocat-wave{0%,100%{transform:rotate(0)}20%,60%{transform:rotate(-25deg)}40%,80%{transform:rotate(10deg)}}@media (max-width:500px){.github-corner:hover .octo-arm{animation:none}.github-corner .octo-arm{animation:octocat-wave 560ms ease-in-out}}</style>
@@ -20,7 +20,7 @@
       <a href="http://codein.withgoogle.com">
         <img class="gci-logo" src="images/logos/gci.png" alt="Code-In logo">
       </a>
-      <h1 data-i18n="page-title">Google Code-in 2017 Current Leaders</h1>
+      <h1 data-i18n="page-title">Google Code-in {{year}} Current Leaders</h1>
     </div>
     <!-- Copyright (c) 2016 Tim Holman (https://github.com/tholman/github-corners)-->
     <div class="ribbon">

